config/environments/production.rb

```diff
@@ -110,4 +110,20 @@
     "https://get-into-teaching-api-dev.london.cloudapps.digital/api"
   config.x.google_maps_key = ENV["GOOGLE_MAPS_KEY"].presence || \
     Rails.application.credentials.google_maps_key.presence
+
+  # Configure Semantic Logging for production environments
+  # This cannot be conditionally loaded so we use it all the time in production
+  # like environments.
+  #
+  # The rails_semantic_logger gem overwrites the log initializer code and by
+  # this point its too late to monkey patch that
+  STDOUT.sync = true
```

Comment:
Why does this need to be synchronous?

Reply:
My experience on other stdout logging is that there can be a significant lag flushing. If the log msg relates to a problem and the service is restarted/node recycled than you never end up with the log info? This is stdout so I wasn't anticipating it being a significant block?

No strong feelings though?

Reply:
Cool, no I'm fine with that, I was just interested in the reason